src/FhirLoader.Common/BundleClient.cs

```diff
@@ -49,6 +49,8 @@ public async Task Send(ProcessedBundle bundle, Action<int, long>? metricsCallbac
             var content = new StringContent(bundle.BundleText!, Encoding.UTF8, "application/json");
             HttpResponseMessage response;
 
+            var requestUri = !string.IsNullOrEmpty(bundle.BundleUri) ? $"/{bundle.ResourceType}?url={bundle.BundleUri}" : string.Empty;
```

Comment:
requestUrl should be:
- `/` for posting of bundles
- `/*resourceType*` for resources that don't have an id
- `/*resourceType*/*id*` for resources that do have an id

Reply:
For posting bundle, in command we have "/" at the end of FHIR Url.
In existing implementation requestUri was empty string for bundle, we have kept it as it in case of bundle.
We have not encountered a scenario where we don't have Id for resource, could you please share example.
For posting profile resources, requestUri would be "/resourceType".
Referring to https://learn.microsoft.com/en-us/azure/healthcare-apis/azure-api-for-fhir/store-profiles-in-fhir

Reply:
- Profiles should generally always be PUT commands to `/*resourceType*/*id*
- Other "single resources" could be PUT to `/*resourceType*/*id* or POST to `/*resourceType*` if there is no ID
- Bundles are a special case and are always POST to `/`,
  - Fun fact, if you POST a bundle to `/Bundle` it will not actually be processed but stored instead.